pkg/agent/cniserver/ipam/antrea_ipam_test.go

```diff
@@ -49,6 +50,8 @@ var (
 	testPear           = "pear"
 	testNoAnnotation   = "empty"
 	testJunkAnnotation = "junk"
+
+	CNIVersion = "0.4.0"
```

Comment:
It shouldn't be public unless there is a good reason.
And perhaps name the varible testCNIVersion

Reply:
rename to testCNIVersion